micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,203 @@
+
+- Removed the extra margin below in inbox search of ws module
+
+## [1.8.40] - [25-Aug-2025]
+
+- Toast error message width issue in Property Tax module(sandbox)
+  
+## [1.8.39] - [19-Aug-2025]
+
+- Removed extra margin on employee pages in the sandbox
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)